src/control/SConscript

```diff
@@ -122,6 +122,9 @@ def scons():
 
     denv = env.Clone()
 
+    if denv.get("COMPILER") == 'covc':
```

Comment:
Will doing this exclude covc for Go build only?
Is this component the last one to be built in the pipeline?
Can you check to make sure the rest of the build is run with covc?
There is a tool cov01 you can use to turn coverage on and off for certain parts of build.

Reply:
yes.  The denv is created on line 123 so this only affects stuff below this SConscript file

Reply:
Thanks